tests/chocolatey-tests/choco-install.Tests.ps1

```diff
@@ -1392,4 +1392,38 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
             $result2.Lines | Should -Contain "Circular dependency detected 'circulardependency2 0.0.1 => circulardependency1 0.0.1 => circulardependency2 0.0.1'."
         }
     }
+
+    Context "Installing package that extracts local zip archive while disabling logging" {
+        BeforeAll {
+            Restore-ChocolateyInstallSnapshot
+            $null = Invoke-Choco source enable -n local

```

Comment:
Is this needed for the tests? Or is this merely related to running the tests without the package being in the internal repository?

Reply:
Good catch, no it is not needed in the tests themself, only is only needed until we have the nupkg file available internally.